Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM node:12-alpine
 COPY . .
 
 RUN npm install && \
-    npm run build && \
-    npm install -g serve
+    npm run build
```

Comment:
what's the point of `npm run build` if you are using `npm run start`

Reply:
When the container starts, the pipeline sets env vars which contain urls for the backend service & ERD services. `npm run build` seems to be the only way to make npm read env vars during container start.